CMakeLists.txt

```diff
@@ -18,12 +21,6 @@ endif()
 #
 # Set project options
 #
-if(${PROJECT_NAME}_IS_TOPLEVEL_PROJECT)
-    set(${PROJECT_NAME}_WARNINGS_AS_ERRORS ON CACHE BOOL "" FORCE)
-else()
-    set(${PROJECT_NAME}_ENABLE_UNIT_TESTING FALSE CACHE BOOL "" FORCE)
-endif()
```

Comment:
(These now get defaulted to ON/OFF later on based on being top-level, rather than unconditionally forced on/off here)